### PR TITLE
Revise

### PR DESCRIPTION
**Pull request by zhekson1**
lil edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,13 +321,13 @@ As a bonus, **the very nature of *illiquidity* implies great arbitrage opportuni
...
+:pushpin: Currently, Plutus is limited in that a spending script is always executed once for *every* script input, **even if the inputs come from the same script address.** So if there are 5 inputs from one of the script addresses, the spending script will be executed 5 times. **The latter 4 are completely redundant in this situation.** These redundant executions impact the maximum number of inputs and outputs that can fit in one transaction. There is a Cardano Problem Statement (CPS) looking to address this ([here](https://github.com/cardano-foundation/CIPs/pull/418)). Nonetheless, basic benchmarking results are shown here:
```
Comment (fallen-icarus) [inline README.md:330]: I don't think this should have an emoji.

File: README.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ Cardano-Swaps achieves batcher/router-free scalability *with* delegation control
...
+The challenge now becomes one of indexing: how do users differentiate each others' swap addresses from all other addresses on Cardano, *without* relying on a specialized indexer/router? This is where Beacon Tokens come into play.
 
 ### Beacon Tokens
 Beacon Tokens are a (WIP) native token standard that "tag" on-chain data in a way that is efficiently queryable by off-chain APIs.  They enable cardano-swaps users to designate their script addresses as "swappable", such that they stand out in sea of other addresses. DDOS/bloat prevention is achieved by carefully marrying Beacons' minting policies with scripts' spending policies. This is expanded upon in the [Specification section](#specification) below.
 
-:notebook: the novel use of *Beacon Tokens* for "tagging" on-chain data can be generalized for many dApps, not just DEXes. More on this in the [Beacon Token CIP](https://github.com/cardano-foundation/CIPs/pull/466)
+:white_check_mark: the novel use of *Beacon Tokens* for "tagging" on-chain data can be generalized for many dApps, not just DEXes. More on this in the [Beacon Token CIP](https://github.com/cardano-foundation/CIPs/pull/466)
```
Comment (fallen-icarus) [inline README.md:65]: Needs capitalization and ending period.

**Comment by fallen-icarus** [comment]
Don't get fancy with emojis. It makes it look noisy, especially when you use different kinds. (In the Swap Contract Logic section, you still use a :notebook:)

**Comment by fallen-icarus** [comment]
Do you disagree?

**Comment by zhekson1** [comment]
Ok I won't mix em up too much but I will use red and green ones depending on context.

**Comment by zhekson1** [comment]
Also, gonna add section in Prelim between LPs and programmable swaps to concentrate on shortcomings of existing orderbook DEXes (scalability cap cause central contract + batchers). Will consolidate such mentions into this section.

**Comment by fallen-icarus** [comment]
Make a smaller PR then. I think we need to add the comment about partial matching now. The rest are not that urgent.

**Comment by zhekson1** [comment]
instead of section, just edited what was already there to include mention of current orderbook shortcomings. Emojis simplified. Punctuation corrected.

**Pull request merged**